app/packages/looker/src/worker/decorated-fetch.test.ts

```
@@ -0,0 +1,91 @@
+    const response = await fetchWithLinearBackoff("http://fiftyone.ai");
+
+    expect(response).toBe(mockResponse);
+    expect(global.fetch).toHaveBeenCalledTimes(2);
+  });
+
+  it("should throw an error after max retries when fetch fails every time", async () => {
+    global.fetch = vi.fn().mockRejectedValue(new Error("Network Error"));
+
+    await expect(
+      fetchWithLinearBackoff("http://fiftyone.ai", 3, 10)
+    ).rejects.toThrowError(new RegExp("Max retries for fetch reached"));
```

Biome (1.9.4)

[error] Use a regular expression literal instead of the RegExp constructor.

Regular expression literals avoid some escaping required in a string literal, and are easier to analyze statically.
Safe fix: Use a literal notation instead.


(lint/complexity/useRegexLiterals)